modules/neopx.py

```diff
@@ -129,16 +104,14 @@ def set(self, identifiers, color):
                 if i >= self.count:
                     pub.sendMessage('log', msg='[LED] Error in set pixels: index out of range')
                     print('Error in set pixels: index out of range')
-                    i = self.count-1                
+                    i = self.count-1
                 self.pixels[i] = color
             except Exception as e:
                 print(e)
                 pub.sendMessage('log', msg='[LED] Error in set pixels: ' + str(e))
                 pass
+        pub.sendMessage('serial', ArduinoSerial.DEVICE_LED, identifiers, color)
```

Comment:
This needs to be in a condition, similar to `if Config.get('neopixel', 'i2c'):` in the code above, so you can define whether to use serial coms for this in the config.json.